### ISSUE TITLE
Run a test on the first run of an auto test

### ISSUE THREAD

**Issue by danielcompton** [bensu/doo#71]
Currently when you run `lein doo chrome test` you get this:

``` sh
$lein doo chrome test
Building ...
... done. Elapsed 2.316741598 seconds

;; ======================================================================
;; Testing with Chrome:

Watching paths: ~/Developer/my-app/src, /Users/Daniel/Developer/my-app/test
```

It would probably make sense to kick off a first test once Karma has started instead of waiting for the user to change something. This might need to go into Karma config?

**Comment by bensu** [comment]
Yeah, this is a problem but haven't had the time to look at it further. Kicking off `karma run` directly after `karma server` might lead to race conditions. I'll follow your suggestion and see if `karma server` can be configured to do it on its own.

**Comment by olivergeorge** [comment]
Perhaps it's worth updating the README to make this explicit.  I found this quite confusing... was doubting my setup when the problem was I needed to change a file to see tests run.

**Comment by metametadata** [comment]
Seems to still be not fixed? I've switched from PhantomJS to Karma and was suprised `lein doo ... auto` works differently.

**Comment by metametadata** [comment]
After several weeks can confirm that it's not fixed. It often hangs at this stage:

```
[doo] Started karma server

;; ======================================================================
;; Testing with Chrome-headless:

[doo] Started karma run
Watching paths: ...
```

**Comment by pesterhazy** [comment]
Same problem as @metametadata: `lein doo chrome myprofile auto` frequently just sits there:

```
;; ======================================================================
;; Testing with Chrome:

[doo] Started karma run
Watching paths:
```

This doesn't always happen, but most of the time for me. Could we re-open this issue?

A workaround is to use

```
lein do doo chrome my-profile once, doo chrome my-profile auto
```

**Comment by miikka** [comment]
Let's re-open. I'm using the `do doo ... once, doo ... auto` workaround as well.
